### DISCUSSION TITLE
Minor Usability issue with expand button when reviewing

### DISCUSSION THREAD

**Discussion by konsalex** [community/community#59315] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hey folks, lately I am facing an annoying issue and though of carding this up.

As seen in the video, the tooltip is visible even though the `Expand` button hover is un-intentional, as I wanted to go and press the collapse button.

This happens if there is an expand button for the code in the beginning of the file in PR review screen.

https://github.com/community/community/assets/12672541/ca61515a-2392-4f1c-b7c8-43bcbba40fb4

**Comment by zbjornson** [comment]
This is a duplicate of https://github.com/orgs/community/discussions/54674.
